src/stars.css

```diff
@@ -0,0 +1,40 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('images/star-active.svg');
+}
```

Comment:
The selector `.stars--0 .stars__star:nth-child(-n + 0)` will not apply any styles because `nth-child(-n + 0)` does not select any elements. If the intention is to have no active stars, this line can be removed or adjusted to reflect that intention.